lib/ai/tools/mcp.ts

```
@@ -1,89 +1,119 @@
-import { experimental_createMCPClient as createMCPClient } from 'ai';
 import * as meetingBaas from '@/server/meetingbaas';
+import { experimental_createMCPClient as createMCPClient } from 'ai';
 
 // Keep track of active clients
 type MCPClientType = Awaited<ReturnType<typeof createMCPClient>>;
 let publicClient: MCPClientType | null = null;
 let privateClient: MCPClientType | null = null;
+let docClient: MCPClientType | null = null;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'docClient' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)